public/js/clients/firefox.js

```diff
@@ -102,7 +102,6 @@ function initPage(actions) {
 
   setupCommands({ threadClient, tabTarget, debuggerClient });
 
-  tabTarget.on("will-navigate", actions.willNavigate);
```

Comment:
Does it only fire twice when you are reloading when it's paused" `frame-update` should not fire if you are reloading a page when it's not paused.